content/en/platform/corda/5.3/deploying-operating/config-users/creating-users.md

```diff
@@ -174,3 +174,99 @@ Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f
 {{< note >}}
 Currently, Corda checks if a user can execute `startFlow` REST operations. No checks are made to whether the user can start a particular flow.
 {{< /note >}}
+
+## Managing User Properties
+
+You can add, list, or remove user properties, as well as find users with a specified property using the endpoints listed in this section. Property is a key-value pair, for example, `postcode`: `SW1A 1AA`. You can have a set of standard keys such as `First Name`, `Middle Names`, `Alternative e-mail address`, `Mobile`, as well as custom keys which only apply for a particular user.
```

Comment:
The standard key set wasn't actually implemented so I think it would be best to remove the part about standard keys. Currently there are no hardcoded key-value pairs.

Reply:
Removed